api/environments/models.py

```diff
@@ -358,6 +375,187 @@ def get_feature_state(
             )
         )
 
+    def get_metrics_payload(self, with_workflows: bool = False) -> EnvMetricsPayload:
+        """
+        Returns total feature count and enabled-by-default feature count
+        scoped to this environment's project.
+        """
+        from edge_api.identities.models import EdgeIdentity
```

Comment:
Can we import from top level?

Reply:
I looked into moving it up, but there’s a circular import between the modules, so keeping it local seemed the best option